cli/common_flags.go

```diff
@@ -84,5 +84,8 @@ func addCommonFlags(flagSet *pflag.FlagSet) *container {
 	// cgroup
 	flagSet.StringVarP(&c.cgroupParent, "cgroup-parent", "", "default", "Optional parent cgroup for the container")
 
+	// disk quota
+	flagSet.StringSliceVar(&c.diskQuota, "disk-quota", nil, "Set disk quota for container")
+
```

Comment:
Could you also give examples in description，like `Set disk quota for container(dir=size)`

Reply:
I will add test for this.

Reply:
PTAL @Letty5411